plugin.json

```diff
@@ -1,22 +1,22 @@
 {
-  "author":"Daniel Hindi"
-  ,"pluginName":"Plugin Name"
-  ,"pluginDescription":"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."
-  ,"supportEmail":"support@buildfire.com"
-  ,"pluginIcon":"icon.png"
-  ,"licenseKey": "a4sd6-as654asd6as545a65s4d6a-65asd564a"
-  ,"control":{
-    "content":{
-      "enabled":true
-    }
-    , "design":{
-      "enabled":true
-    }
-    , "settings":{
-      "enabled":false
-    }
+  "author":"abhay chauhan"
+,"pluginName":"contact us"
+,"pluginDescription":"This is a nice contact us plugin"
+,"supportEmail":"abhay.singh@rootinfosol.com "
+,"control":{
+  "content":{
+    "enabled":true
   }
-  ,"widget":{
-
+, "design":{
+    "enabled":true
+  }
+, "settings":{
+    "enabled":false
   }
+}
+,"widget":{
+
+},
+  "features" : [],
+  "languages" : ["en"]
```

Comment:
on the images in the resource solder. 1. why are these add 2.no dependencies should be in resources bc that folder wont be carried over to the app only the widget folder